apps/dashboard/src/app/(dashboard)/interest-group/[id]/edit-card.tsx

```diff
@@ -33,11 +33,23 @@ export const InterestGroupEditCard: FC = () => {
     defaultValues: interestGroup,
   })
   return (
-    <div>
+    <Stack gap="xs">
       <FormComponent />
-      <Button variant="outline" color="red" style={{ marginTop: "1rem" }} onClick={open}>
-        <Icon icon="tabler:trash" />
-      </Button>
-    </div>
+      <div style={{ display: "flex", flexWrap: "wrap", alignItems: "center"}}>
+        <Button variant="outline" color="red" style={{ width: "55px", marginRight: "24px" }} onClick={open}>
+          <Icon icon="tabler:trash" />
+        </Button>
+        <Checkbox label="Aktiv?" onChange={ () =>
+            edit.mutate({
+              id: interestGroup.id,
+              values: {
+                name: interestGroup.name,
+                description: interestGroup.description,
+                link: interestGroup.link,
+                isActive: !interestGroup.isActive,
+              }
+            })} checked={interestGroup.isActive} />
```

Comment:
you should use spread elements to avoid having to define all the mutate values

`                    ...interestGroup,
                    values: {
                        ...interestGroup,
                        isActive: !interestGroup.isActive,
                    },
`

Reply:
Didn't know that was an option, thanks :)